wcivf/apps/elections/views/mixins.py

```diff
@@ -69,6 +69,11 @@ def postcode_to_ballots(self, postcode, compact=False):
         pes = pes.select_related("post")
         pes = pes.select_related("election")
         pes = pes.select_related("election__voting_system")
+
+        # only retrieve referendums until 6th May
+        if date.today() <= date(2021, 5, 6):
```

Comment:
This is ok, but will result in the tests failing after the election. Let's keep this `prefetch_related` in all cases, so the number of queries stays the same, but keep the logic in `get_referendums`